### PR TITLE
Add migrations for favorites  #28

### LINKED ISSUE #28: Databases: Create Favorites Table (Migrations)
## User story

**Who:** **As a** developer
**What:** **I want to** create the FAVORITES table in the Database by using the implementation details.
**Why:** **so that I can** add new FAVORITES relations.

## Acceptance criteria

FAVORITES table has been created with all the required fields. 
Screenshot 2022-01-25 at 21 00 25

### PR DESCRIPTION
**Pull request by nataliasowa**
# Description

Databases: Create Migrations for Favourites Table

Fixes #28 

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by h09shais** [comment]
Nice effort @NataliaS1994  !
Have you tried using `db:setup` from `package.json`?
Does `db:setup` follow your changes successfully :) ?

**Comment by nataliasowa** [comment]
Thank you @h09shais. I tried to npm run db:setup but I received an error that no user_id found. I think it's because there is no user database delivered yet.

**Comment by h09shais** [comment]
Create the user table and set the relation then 😀!

File: src/server/migrations/20200510015322_create_modules_schema.js [hunk 1]
```diff
@@ -1,18 +0,0 @@
-// TODO: This is a sample migration for demonstration. Remove before adding real ones.
-
```
Comment (senner007) [inline src/server/migrations/20200510015322_create_modules_schema.js]: See here for the discussion about the modules tables #18

Either remove the backend endpoint as well or keep everything including migrations and seeds

**Review by senner007 — CHANGES_REQUESTED** [review]

**Review by pankaj28843 — APPROVED** [review]
Tested on my machine, works well!

**Pull request merged**